foo.out

```diff
@@ -0,0 +1 @@
+hello world
```

Comment:
Why do we have `foo.error` and `foo.out` files committed to the repo?

Reply:
95efe852ac5976e47000b00ede3f254b0b92a091